src/TypeSpec.Extension/Emitter.Csharp/src/constants.ts

```diff
@@ -7,3 +7,4 @@ export const projectedNameClientKey = "client";
 export const mockApiVersion = "0000-00-00";
 export const tspOutputFileName = "tspCodeModel.json";
 export const configurationFileName = "Configuration.json";
+export const packageName = "@azure-tools/typespec-csharp";
```

Comment:
I think it should be 
```
require('./package.json').name;
```

Reply:
@pshao25 We cannot use that because of the typescript setting. We're using ES module which doesn't support `require`: 
> error file-load: require is not defined in ES module scope, you can use import instead